pontoon/base/models.py

```diff
@@ -2316,6 +2316,25 @@ def rejected(self, locale, project=None):
             )
         )
 
+    def missing_no_suggestions(self, locale, project=None):
+        """Return a filter to be used to select entities with no or only rejected translations.
+
+        This filter will return all entities that have no or only rejected translations.
+        :arg Locale locale: a Locale object to get translations for
+
+        :returns: a django ORM Q object to use as a filter
+
+        """
+        return ~Q(
+            pk__in=self.get_filtered_entities(
+                locale,
+                Q(approved=True) | Q(rejected=False),
+                lambda x: x.approved and not x.rejected,
+                match_all=True,
```

Comment:
`False`: even if one plural form has missing strings only, the filter should catch it.

Reply:
`True` in this case will get us entity ids for which has an approved or unreviewed string is present for "all" plurals (not of this will be at least one) . So I think this is correct. 

I started with False as first instinct but testing it out with True gives the right behaviour

Reply:
Good point! Note that you can also remove the attribute (to be consistent with the `missing()` filter we're mimicing here), because the default value is `True`.